### DISCUSSION TITLE
mime types tab desperately empty

### DISCUSSION THREAD

**Discussion by audioscavenger** [allinurl/goaccess#2663] [Q&A] [answered]
nginx returns mimes in that format: "application/json; charset=UTF-8"
how do we get goaccess to recognize it?

**Comment by allinurl** [comment, marked as answer]
You should be able to grab it with `%M`. For `"application/json; charset=UTF-8"`, just `%M, %^"` should work.

**Reply by audioscavenger** [reply]
AAAHHH that's smart, thanks
